bin/pullpreview

```diff
@@ -76,12 +94,14 @@ begin
     end
 
     PullPreview::SyncWithGithub.run(app_path, opts.to_hash)
+  when "license-create"
```

Comment:
Options should be parsed here. The classes are not supposed to know about the CLI, especially if we want to reuse the class from another ruby program.